BrainPortal/app/helpers/documentation_helper.rb

```diff
@@ -32,11 +32,12 @@ module DocumentationHelper
   # shown documentation page.
   # See the help_document views for more details.
   def help_button(key, display = "Help")
-    doc = HelpDocument.find_by_key(key) || HelpDocument.from_existing_file!(key)
+    doc = HelpDocument.find_by_key(key)
```

Comment:
I don't know why the `HelDocument.from_existing_file!(key)` disappeared, and what did it do.